main.go

```diff
@@ -16,14 +16,17 @@
 package main
 
 import (
+	"context"
+
 	"github.com/hazelcast/hazelcast-commandline-client/commands"
 )
 
 func main() {
+	ctx := context.Background()
 	isInteractive := commands.IsInteractiveCall()
 	if isInteractive {
-		commands.ExecuteInteractive()
+		commands.ExecuteInteractive(ctx)
```

Comment:
You can `return` after this line to be able to remove the `else` in the line below.

Reply:
That's better thanks, fixed at https://github.com/hazelcast/hazelcast-commandline-client/pull/40/commits/e5b6f7a183de2c648671a2ffa62fad00b79baf62